src/box/tuple.h

```diff
@@ -789,6 +789,29 @@ tuple_format(struct tuple *tuple)
 	return format;
 }
 
+static inline struct tuple_format *
+tuple_field_map_format(const char *tuple_data)
+{
+	field_map_format_id_t fmf_id;
+	if ((intptr_t)tuple_data % 4 == 2)
+		fmf_id = 0;
```

Comment:
It would be nice to have something like `FORMAT_ID_DEFAULT = 0` for the sake of maintainability.